config.yaml

```diff
@@ -8,8 +8,15 @@
 # lc: Library Carpentry
 # cp: Carpentries (to use for instructor training for instance)
 # incubator: The Carpentries Incubator
+# 
+# NB: this option supports custom types so lessons can be branded
```

Comment:
```suggestion
# This option supports custom types so lessons can be branded
```

Reply:
Removing "NB:" as it may be confusing to folks who may think that they should put NB as the carpentry value.